icu4c/source/i18n/calendar.cpp

```diff
@@ -4247,14 +4247,11 @@ Calendar::internalSet(EDateFields field, int32_t value)
     internalSet(static_cast<UCalendarDateFields>(field), value);
 }
 
-int32_t Calendar::internalGetMonth(UErrorCode& status) const {
-    if (U_FAILURE(status)) {
-        return 0;
-    }
+int32_t Calendar::internalGetMonth(UErrorCode& /* status */ ) const {
     if (resolveFields(kMonthPrecedence) == UCAL_MONTH) {
-        return internalGet(UCAL_MONTH, status);
+        return internalGet(UCAL_MONTH);
```

Comment:
Are you sure this is what you want to do?  I thought I remembered there was another overload of `internalGet` that took _three_ parameters, the third of which was `status`.  Would it make more sense to use that one, with a dummy value for the second parameter, or does that not make sense (or waste time getting the same result?

Reply:
I think I was thinking of this:

```
int32_t Calendar::internalGetMonth(int32_t defaultValue, UErrorCode& /* status */) const {
    if (resolveFields(kMonthPrecedence) == UCAL_MONTH) {
        return internalGet(UCAL_MONTH, defaultValue);
    }
    return internalGet(UCAL_ORDINAL_MONTH);
}
```

So does it make sense for the version of `internalGetMonth()` that Frank is changing to just call the other version?

Then again, since that other version isn't doing anything with the error code either, maybe it doesn't really make a difference...